proposed/simplecache.md

```diff
@@ -0,0 +1,230 @@
+     *
+     * @param string $key The unique cache key of the item to delete
+     *
+     * @return void
+     */
+    public function delete($key);
+
+    /**
+     * Wipe clean the entire cache's keys
+     *
+     * @return void
+     */
+    public function clear();
+
+    /**
+     * Obtain multiple cache items by their unique keys
+     *
+     * @param array|Traversable $keys A list of keys that can obtained in a single operation.
+     *
+     * @return array An array of key => value pairs. Cache keys that do not exist or are stale will have a value of null.
+     */
+    public function getMultiple($keys);
+
+    /**
+     * Persisting a set of key => value pairs in the cache, with an optional TTL.
+     *
+     * @param array|Traversable     $items An array of key => value pairs for a multiple-set operation.
+     * @param null|int|DateInterval $ttl   Optional. The amount of seconds from the current time that the item will exist in the cache for.
+     *                                     If this is null then the cache backend will fall back to its own default behaviour.
+     *
+     * @return bool True on success and false on failure
+     */
+    public function setMultiple($items, $ttl = null);
+
+    /**
+     * Delete multiple cache items in a single operation
+     *
+     * @param array|Traversable $keys The array of string-based keys to be deleted
+     *
+     * @return void
+     */
+    public function deleteMultiple($keys);
+
+    /**
+     * Identify if an item is in the cache.
+     * NOTE: It is recommended that exists() is only to be used for cache warming type purposes
+     * and not to be used within your live applications operations for get/set, as this method
+     * is subject to a race condition where your exists() will return true and immediately after,
+     * another script can remove it making the state of your app out of date.
+     *
+     * @param string $key The cache item key
+     *
+     * @return bool
+     */
+    public function exists($key);
+
+}
+```
+
+### 2.2 CounterInterface
+
+For counters it provides the ability to increment and decrement a cache key atomically.
+
+``` php
+<?php
+namespace Psr\SimpleCache;
+
+interface CounterInterface
+{
+    /**
+     * Increment a value atomically in the cache by its step value, which defaults to 1
+     *
+     * @param string  $key  The cache item key
+     * @param int $step The value to increment by, defaulting to 1
+     *
+     * @return int|bool The new value on success and false on failure
+     */
+    public function increment($key, $step = 1);
+
+    /**
+     * Decrement a value atomically in the cache by its step value, which defaults to 1
+     *
+     * @param string  $key  The cache item key
+     * @param int $step The value to decrement by, defaulting to 1
+     *
+     * @return int|bool The new value on success and false on failure
+     */
+    public function decrement($key, $step = 1);
+}
+```
```

Comment:
> inc/dec on non-existing key should return 1/-1 IMO. If this isn't clear
we should clarify.

perfect, updating the description would help I think as I read the defaults are only applied on the `step` argument.

I think raising an exception when the boundaries are crossed would be nice, but the implementation for this would be harder/introduces overhead. We could leave the handing for max./min. value up to the users of the interface; so defining that the behavior is undefined would be OK with me :)